### DISCUSSION TITLE
Implement Flux ControlNet functionality

### DISCUSSION THREAD

**Discussion by softtaco1** [lllyasviel/stable-diffusion-webui-forge#1068] [Ideas]
There are already a few models out there to use with FLUX that currently do not work. 

(See https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1043 & https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/1067 for details)

I know there is lots in the work already, but it would be awesome to see ControlNets for Flux being usable on Forge!

Thank you for reading and thank you lllyasviel for providing this wonderful tool to the community.

**Comment by LOR-Studio** [comment]
I am also anticipating this feature with Flux. I'm using NF4 models currently, but it does not seem to matter which kind I use or the Controlnet model. I can however generated previews with Flux Controlnet, but when attempting to generate an image it receives this error: 

```
*** Error running postprocess_batch_list: D:\AI_Tools\Stable_Diffusion\stable-diffusion-webui-forge\extensions-builtin\sd_forge_controlnet\scripts\controlnet.py
    Traceback (most recent call last):
      File "D:\AI_Tools\Stable_Diffusion\stable-diffusion-webui-forge\modules\scripts.py", line 916, in postprocess_batch_list
        script.postprocess_batch_list(p, pp, *script_args, **kwargs)
      File "D:\AI_Tools\Stable_Diffusion\stable-diffusion-webui-forge\venv\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
        return func(*args, **kwargs)
      File "D:\AI_Tools\Stable_Diffusion\stable-diffusion-webui-forge\extensions-builtin\sd_forge_controlnet\scripts\controlnet.py", line 567, in postprocess_batch_list
        self.process_unit_after_every_sampling(p, unit, self.current_params[i], pp, *args, **kwargs)
    KeyError: 0
```

It suggests something is wrong with self.current_params, though it may be outside my scope of understanding...as the issue could be more complex. Maybe this helps...in any case keep up all your hard work lllyasviel!

**Comment by mondermurshed** [comment]
Did you changed your UI After not receiving any fix for this problem ?
